dali/base/dautils.cpp

```diff
@@ -119,7 +119,19 @@ IPropertyTree * getDropZonePlane(const char * name)
 
 bool isPathInPlane(IPropertyTree *plane, const char *path)
 {
-    return isEmptyString(path) || startsWith(path, plane->queryProp("@prefix"));
+    if (isEmptyString(path))
+        return true;
+
+    StringBuffer prefix(plane->queryProp("@prefix"));
+    const char *p = addPathSepChar(prefix).str(); //Add a trailing path separator to the prefix if it does not have one.
```

Comment:
I think the copy and addPathSepChar involved here are unnecessary steps/overhead.

startsWith is correct, *except* that we don't want to return true from this function, unless it has exactly matched the path (startsWith reached end of path), or the next character in path is '/'. Or, it reached end of path, and prefix is at a trailing  '/'.
You could write a variant of startsWith that returned the position in path and then test these conditions, but not sure it's worth a new method, probably better to implement directly here instead.